containers/2_ApplicationSpecific/abaqus/README.md

```diff
@@ -1,28 +1,48 @@
 # Running Abaqus with Apptainer
 
-Abaqus 2024 is available via an Apptainer/Singularity container. You can find the Abaqus container here: `/util/software/containers/x86_64/abaqus-2024.sif`
+Abaqus 2024 is available via an Apptainer/Singularity container. The Abaqus container file can be found in this directory: `/util/software/containers/x86_64/abaqus-2024.sif` which is accessible when logged into CCR's HPC environment.
 This is a large file (10GB) so please be sure to copy it to a location where you have enough space (i.e. your project directory), or you can run the container from this location.
 
 **Note**: Apptainer/Singularity is only available on compute and compile nodes. Please refer to our [container documentation](https://docs.ccr.buffalo.edu/en/latest/howto/containerization/) for more information on using containers.
 
 ## Abaqus Command Line/Batch Script
 
-To run Abaqus from the command line, run this command: `$ apptainer exec -B /util:/util,/scratch:/scratch /util/software/containers/x86_64/abaqus-2024.sif /bin/bash`
+There are several options to run Abaqus from the command line. If you're unsure of the options to provide or want to run the application more interactively, use the `shell` option with Apptainer. This will drop you into the container and then you can work with Abaqus interactively. For example:
+```
+apptainer shell -B /util:/util,/scratch:/scratch /[path-to-container]/abaqus-2024.sif /bin/bash
```

Comment:
My bad - I forgot apptainer doesn't need the path to the shell when using the shell option
```suggestion
apptainer shell -B /util:/util,/scratch:/scratch /[path-to-container]/abaqus-2024.sif
```